tests/live_pixel_validation_test.mjs

```diff
@@ -115,6 +115,7 @@ describe('Common suffixes', () => {
                     enum: [1, 2, 3],
                 },
             ],
+            parameters: [],
```

Comment:
We should update the [schema](https://github.com/duckduckgo/pixel-schema/blob/main/schemas/pixel_schema.json5#L13) itself then if we'll be requiring 'parameters' to be defined.

Reply:
Great catch! I just updated the PR to support empty params instead, I think easier?